src/tool/subcommands/api_cmd.rs

```diff
@@ -38,6 +39,17 @@ pub enum NodeType {
     Lotus,
 }
 
+/// Report mode for the API compare tests.
+#[derive(Debug, Clone, Copy, ValueEnum)]
+pub enum ReportMode {
+    /// Show everything (default)
+    Full,
```

Comment:
If it's the default, let's make it really a default. Some more changes might be needed elsewhere.
```suggestion
#[derive(Debug, Default, Clone, Copy, ValueEnum)]
pub enum ReportMode {
    /// Show everything
    #[default]
    Full,
```

Reply:
Removed the `default` from here, since this is coming from flag (the flag should accept the default value).